docs/core/linux-prerequisites.md

```diff
@@ -103,537 +105,288 @@ On Linux, there are two installer package choices:
 
 # [.NET Core 2.x](#tab/netcore2x)
 
-2. Install system components.
-
-   ```bash
-   sudo apt-get update
-   sudo apt-get install curl libunwind8 gettext apt-transport-https
-   ```
-   
-3. Register the trusted Microsoft Product key.
-
-   ```bash
-   curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.gpg
-   sudo mv microsoft.gpg /etc/apt/trusted.gpg.d/microsoft.gpg
-   ```
-   
-4. Register the Microsoft Product feed.
-
-   **Debian 9 (Stretch)**
-
-   ```bash
-   sudo sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-debian-stretch-prod stretch main" > /etc/apt/sources.list.d/dotnetdev.list'
-   ```
-   
-   **Debian 8 (Jessie)**
-   
-   ```bash
-   sudo sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-debian-jessie-prod jessie main" > /etc/apt/sources.list.d/dotnetdev.list'
-   ```
-   
-5. Install .NET Core SDK.
-
-   ```bash
-   sudo apt-get update
-   sudo apt-get install dotnet-sdk-2.0.0
-   ```
-
-6. Add dotnet to your PATH.
-
-   ```bash
-   export PATH=$PATH:$HOME/dotnet
-   ```
-   
-7. Run the `dotnet --version` command to prove the installation succeeded.
-
-   ```bash
-   dotnet --version
-   ```   
-  
-
-# [.NET Core 1.x](#tab/netcore1x)
+1. Remove any **previous preview** versions of .NET Core from your system.
 
-2. Get the prerequisites.
+2. Install .NET Core 2.x on supported Debian versions (64 bit):
 
-   ```bash
-   sudo apt-get install curl libunwind8 gettext
-   ```
+**.NET Core 2.0**
 
-3. Download the .NET Core SDK binaries (tarball).
+|Runtimes / SDKs          |Debian 9       |Debian 8       |
+|-------------------------|---------------|---------------|
+|.NET Core Runtime 2.0.6  |[Install link](https://www.microsoft.com/net/download/linux-package-manager/debian9/runtime-2.0.6)   |[Install link](https://www.microsoft.com/net/download/linux-package-manager/debian8/runtime-2.0.6)   |
+|.NET Core Runtime 2.0.5  |[Install link](https://www.microsoft.com/net/download/linux-package-manager/debian9/runtime-2.0.5)   |[Install link](https://www.microsoft.com/net/download/linux-package-manager/debian8/runtime-2.0.5)   |
+|.NET Core SDK 2.1.103    |[Install link](https://www.microsoft.com/net/download/linux-package-manager/debian9/sdk-2.1.103)   |[Install link](https://www.microsoft.com/net/download/linux-package-manager/debian8/sdk-2.1.103)   |
+|.NET Core SDK 2.0.3      |[Install link](https://www.microsoft.com/net/download/linux-package-manager/debian9/sdk-2.0.3)   |[Install link](https://www.microsoft.com/net/download/linux-package-manager/debian8/sdk-2.0.3)   |
 
-   ```bash
-   curl -sSL -o dotnet.tar.gz https://go.microsoft.com/fwlink/?linkid=848826
-   ```
+**.NET Core 2.1**
 
-4. Extract the .NET Core SDK binaries.
+>[!IMPORTANT]
+> To use .NET Core 2.1 with Visual Studio, you'll need to [install Visual Studio 2017 15.7 Preview 1 or newer](https://www.visualstudio.com/vs/preview/?dotnetid=169771757.1498243949).
```

Comment:
remove future tense and do you need that dotnetid there? should you mention when to expect support for VS for Mac (see https://blogs.msdn.microsoft.com/dotnet/2018/04/11/announcing-net-core-2-1-preview-2/)?

@TerryGLee is there any special campaign link for the preview version?